src/apps/mesoscale-explorer/ui/entities.tsx

```diff
@@ -229,6 +242,46 @@ export class SelectionInfo extends PluginUIComponent<{}, { isDisabled: boolean }
     }
 }
 
+
+export class FocusInfo extends PluginUIComponent<{}, { isDisabled: boolean }> {
+    state = {
+        isDisabled: false,
+    };
+
+    componentDidMount() {
+        this.subscribe(this.plugin.state.data.behaviors.isUpdating, v => {
+            this.setState({ isDisabled: v });
+        });
+
+        this.subscribe(this.plugin.managers.structure.selection.events.changed, e => {
+            if (!this.state.isDisabled) {
+                this.forceUpdate();
+            }
+        });
+    }
+
+    get info() {
+        // const infos: { label: string, key: string, description?: string }[] = [];
+        const info: { focusInfo: string } = { focusInfo: '' };
```

Comment:
Return just the `focusInfo` string since that is the only property?

Reply:
Thanks @dsehnal I simplified the function in the last commit. The component is at the bottom right panel, if you think there is a better place for it let me know. I like it here as it help isolated the entity in the tree.